### ISSUE TITLE
System Error when running Ganache 2.1.2 on win32

### ISSUE THREAD

**Issue by shravangs56** [trufflesuite/ganache#1492]
PLATFORM: win32
GANACHE VERSION: 2.1.2

EXCEPTION:
```
Error: EPERM: operation not permitted, rename 'C:\Users\ssrinivas35\AppData\Roaming\Ganache\workspaces\symptomatic-pizzas\Settings.1186691046' -> 'C:\Users\ssrinivas35\AppData\Roaming\Ganache\workspaces\symptomatic-pizzas\Settings'
    at Object.fs.renameSync (fs.js:772:18)
    at writeFileSync (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\write-file-atomic\index.js:124:8)
    at LocalStorage.setItem (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\node_modules\node-localstorage\LocalStorage.js:188:7)
    at JsonStorage.setToStorage (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\types\json\JsonStorage.js:72:20)
    at JsonStorage.set (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\types\json\JsonStorage.js:78:12)
    at WorkspaceSettings.set (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\types\settings\Settings.js:89:21)
    at Workspace.saveAs (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\types\workspaces\Workspace.js:91:21)
    at EventEmitter.<anonymous> (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\main.js:547:24)
    at Generator.next (<anonymous>)
    at step (C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\main.js:71:191)
    at C:\Program Files\WindowsApps\Ganache_2.1.2.0_x64__zh355ej5cj694\app\resources\app.asar\src\main\main.js:71:361
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by davidmurdoch** [comment]
Should be fixed in https://github.com/trufflesuite/ganache/releases/tag/v2.3.0! Please let me know if you are still experiencing problems!

**Comment by wbt** [comment]
For those tracing through...this [was not](https://github.com/trufflesuite/ganache/issues/1721) fixed in 2.3.0.
